camunda-utils/Camunda/Camunda.py

```diff
@@ -8,6 +8,22 @@
 from typing import Any, List, Optional, Union, Dict
 from robot.api.deco import keyword
 from robot.api import logger
+from robot.libraries.BuiltIn import BuiltIn
+
+SECRET_VARIABLE = "${secrets}"
+SECRET_ENV_NAME = "CAMUNDA_SECRETS"
+
+
+class Secrets(dict):
+    """
+    A wrapper around a dictionary that allows access to the values via attributes.
+    This allows access as `${SECRETS.key}` instead of `${SECRETS['key']}`
+    """
+
+    def __getattr__(self, name):
+        if name in self:
+            return self[name]
+        raise AttributeError(f"'Secrets' object has no attribute '{name}'")
```

Comment:
There is an inconsistency here when the secret is not in the dictionary. If using the attribute syntax `${SECRETS.key}` then your AttributeError will be thrown. If using the container syntax `${SECRETS['Name-With-Dash']}` the usual KeyError will be thrown instead.